mysql/mysql-lib.pl

```diff
@@ -1304,6 +1304,37 @@ sub mysql_size_input
 		    [ "M", "MB" ], [ "G", "GB" ] ]);
 }
 
+# get_table_index_stats(db)
+# Retrieves index stats for all tables in the given database
+sub get_table_index_stats
+{
+my ($db) = @_;
+my $table_list = join(", ", map { "'$_'" } &list_tables($db));
+my $sql_query = "
+    SELECT 
+        TABLE_SCHEMA,
+        TABLE_NAME,
+        INDEX_NAME,
+        NON_UNIQUE,
+        SEQ_IN_INDEX,
+        COLUMN_NAME,
+        COLLATION,
+        CARDINALITY,
+        SUB_PART,
+        PACKED,
+        NULLABLE,
+        INDEX_TYPE,
+        COMMENT,
+        INDEX_COMMENT
+    FROM 
+        INFORMATION_SCHEMA.STATISTICS
+    WHERE 
+        TABLE_SCHEMA = '$db' AND TABLE_NAME IN ($table_list);
```

Comment:
Use a proper SQL parameter instead of substituting $db into the string

Reply:
Yes, sure! Check this patch https://github.com/webmin/webmin/pull/2234/commits/5210f623f3f1a0b8540fa6ac0d2c94689a4f1f8f.

Reply:
Excellent thanks!